.changeset/eight-windows-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": patch
+---
+
+Fix `useDataGridExcelExport` with undefined values in data grid
```

Comment:
Please explain what exactly was fixed/what was the issue. 
Maybe like this:

```suggestion
Fix a bug in `useDataGridExcelExport` that would cause an Excel export to fail when a cell's value was `undefined`
```